File: messages/src/main/java/com/thundashop/zauiactivity/dto/OctoBookingReserve.java
package com.thundashop.zauiactivity.dto;

import lombok.Data;

import java.util.List;

@Data
public class OctoBookingReserve {
    private String id;
    private String uuid;
    private String supplierReference;
    private String status;
    private String resellerReference;
    private String utcCreatedAt;
    private String utcUpdatedAt;
    private String utcExpiresAt;
    private String utcRedeemedAt;
    private String utcConfirmedAt;
    private String productId;
    private String freesale;
    private String availabilityId;
    private String notes;
    private String optionId;
    private OctoProductAvailability availability;
    private Contact contact;
    private List<String> deliveryMethods = null;
    private List<String> voucher = null;
    private List<UnitItemReserveResponse> unitItems = null;
    private Cancellation cancellation;
}

class UnitItemReserveResponse {
    private String uuid;
    private String unitId;
    private Object resellerReference;
    private Object supplierReference;
    private Ticket ticket;
}
class Contact {
    private String fullName;
    private String firstName;
    private String lastName;
    private String emailAddress;
    private String phoneNumber;
    private List<Object> locales = null;
    private String country;
    private String notes;
}
class DeliveryOption {
    private String deliveryFormat;
    private String deliveryValue;
}

class Ticket {
    private String redemptionMethod;
    private String utcDeliveredAt;
    private String utcRedeemedAt;
    private List<DeliveryOption> deliveryOptions = null;
}
class Cancellation {
    private Boolean cancelled;
    private String utcCancelledAt;
}

File: com.getshop.client/events/zauiactivity/dto/OctoBookingReserve.php
<?php
class zauiactivity_dto_OctoBookingReserve {
	/** @var String */
	public $id;

	/** @var String */
	public $uuid;

	/** @var String */
	public $supplierReference;

	/** @var String */
	public $status;

	/** @var String */
	public $resellerReference;

	/** @var String */
	public $utcCreatedAt;

	/** @var String */
	public $utcUpdatedAt;

	/** @var String */
	public $utcExpiresAt;

	/** @var String */
	public $utcRedeemedAt;

	/** @var String */
	public $utcConfirmedAt;

	/** @var String */
	public $productId;

	/** @var String */
	public $freesale;

	/** @var String */
	public $availabilityId;

	/** @var String */
	public $notes;

	/** @var String */
	public $optionId;

	/** @var zauiactivity_dto_OctoProductAvailability */
	public $availability;

	/** @var zauiactivity_dto_OctoSupplierContact */
	public $contact;

	/** @var String */
	public $deliveryMethods;

	/** @var String */
	public $voucher;

	/** @var class com_thundashop_zauiactivity_dto_UnitItemReserveResponse[] */
	public $unitItems;

	/** @var zauiactivity_dto_Cancellation */
	public $cancellation;

}
?>